### ISSUE TITLE
creek 1.1.2 depends on nokigiri ~> 1.6.0 which has a security vulnerability

### ISSUE THREAD

**Issue by salimane** [pythonicrubyist/creek#41]
The following is the output of bundle audit
```
ruby-advisory-db: 284 advisories
Name: nokogiri
Version: 1.6.8.1
Advisory: CVE-2016-4658
Criticality: Unknown
URL: https://github.com/sparklemotion/nokogiri/issues/1615
Title: Nokogiri gem contains several vulnerabilities in libxml2 and libxslt
Solution: upgrade to >= 1.7.1

Vulnerabilities found!
```

But creek depends on that specific version of nokigiri. can we release a version of creek that depends on nokigiri 1.7.1 ?

Thanks

**Comment by chifung7** [comment]
Need this to fix `warning: constant ::Fixnum is deprecated` for ruby 2.4.0 as well.

**Comment by rubiii** [comment]
Any news on this one?

**Comment by SGrondin** [comment]
@salimane @chifung7 @rubiii 
It's all fixed in my fork at https://github.com/partpilotllc/creek
Just add this line to your Gemfile:
```ruby
gem 'creek', '2.0.0-partpilot', :git => "https://github.com/PartPilotLLC/creek.git", :tag => "v2.0.0-partpilot"
```
There's also a PR for it (https://github.com/pythonicrubyist/creek/pull/47), but I have no idea when/if it'll be merged.

**Comment by pythonicrubyist** [comment]
This issue is fixed in PR #47. And merged to master.
